core/tls-websocket-server/src/ws_server.rs

```diff
@@ -288,15 +292,13 @@ mod tests {
 
 	#[test]
 	fn server_handles_multiple_connections() {
-		let _ = env_logger::builder().is_test(true).try_init();
+		let _ = env_logger::builder().is_test(false).try_init();
 
 		let expected_answer = "websocket server response bidibibup".to_string();
 		let port: u16 = 21777;
-		const NUMBER_OF_CONNECTIONS: usize = 20;
+		const NUMBER_OF_CONNECTIONS: usize = 100;
```

Comment:
This is the test that allowed me to reproduce the problem. The number of connections had to be increased to ~ 600 for the problem to show consistently.
Another issue is, that on Linux there is a limit of file descriptors (e.g. sockets) for each process. By default, that is 1024. Once that limit is reached, we cannot accept more connections (until old connections close again)

Reply:
Can you add this piece of information to a doc somewhere? Maybe to the README of the worker?

Reply:
Yes -> https://github.com/integritee-network/book/pull/32

Reply:
Very cool, thanks! One last request: can you add this to the readme in this repository? I believe this will be a problem pretty much everyone will stumble upon, and the local README is the most visible one.

Reply:
I think that's alright. Our README is not only minimal, but also outdated. But the same goes for the book where you've just added that info. The good thing: This info will most likely not get outdated any time soon.